libvast/test/system/partition_transformer.cpp

```diff
@@ -229,10 +243,101 @@ TEST(delete transform / persist before done) {
       CHECK_EQUAL(synopsis_legacy->id_range()->end(), IDSPACE_BEGIN + events);
     },
     [](const caf::error& e) {
-      FAIL("unexpected error" << e);
+      FAIL("unexpected error " << e);
     });
 }
 
+TEST(partition with multiple types) {
+  // Spawn partition transformer
+  auto store_id = "segment-store"s;
+  auto synopsis_opts = vast::index_config{};
+  auto index_opts = caf::settings{};
+  auto transform = std::make_shared<vast::transform>(
+    "partition_transform"s, std::vector<std::string>{"zeek.conn"});
```

Comment:
The outcome should be the same in this case, but the following should be more correct:
```suggestion
    "partition_transform"s, std::vector<std::string>{});
```